test/configCases/output-module/inlined-module/webpack.config.js

```diff
@@ -11,3 +11,29 @@ module.exports = {
 	},
 	target: "es2020"
 };
+
+/** @type {import("../../../../").Configuration[]} */
+module.exports = [
+	{
+		...base,
+		name: "module-entryIife-false",
+		output: {
+			filename: "module-entryIife-false.mjs"
+		},
+		optimization: {
+			...base.optimization,
+			entryIife: false
+		}
+	},
+	{
+		...base,
+		name: "module-entryIife-true",
+		output: {
+			filename: "module-entryIife-true.mjs"
+		},
+		optimization: {
+			...base.optimization,
+			entryIife: true
+		}
+	}
+];
```

Comment:
@fi3ework Are you sure our test are fine? I tried to run it and the both configurations output the same content...

Try it locally, it is due:
```js
const innerStrict = !allStrict && allModules.every(m => m.buildInfo.strict); // only the single entry for now.
const singleEntryWithModules = inlinedModules.size === 1 && hasChunkModules;
```

That's why I asked to add test cases

Reply:
Due to `inlinedModules.size === 1`, because we have 2 inlined modules, I think we should size

Reply:
We should use `singleEntryWithModules && innerStrict`, also why we don't do it when `inlinedModules.size > 1`? I think we already check this in our code...

Reply:
> We should use singleEntryWithModules && innerStrict

Yes, it's a mistake. I'm updating the PR.

> also why we don't do it when inlinedModules.size > 1

Not fully yet. We checked the through vars in non-inline modules and the module vars in inline modules, and renamed the module var's name there is a collision. That's how the optimization runs now.

But for multiple entries, there also might be a through var in inline module A and a module var in inline module B, which cause the same var leak.

So we can only handle single entry optimization now.

Reply:
> But for multiple entries, there also might be a through var in inline module A and a module var in inline module B, which cause the same var leak.

Let's add a test case for this, to aviod regressions or future optimizations